enterprise/frontend/src/metabase-enterprise/rich_text_editing/tiptap/extensions/CardEmbed/CardEmbedNode.tsx

```diff
@@ -477,6 +644,15 @@ export const CardEmbedComponent = memo(
                         </Flex>
                       </Menu.Target>
                       <Menu.Dropdown>
+                        <Menu.Item
+                          onClick={() => {
+                            commentsPath && dispatch(push(commentsPath));
+                          }}
+                          disabled={!commentsPath}
+                          leftSection={<Icon name="add_comment" size={14} />}
+                        >
+                          {t`Comment`}
+                        </Menu.Item>
```

Comment:
At the very least we should use `Link` here because `Menu` here uses `withinPortal` - there shouldn't be any TipTap interference.

Something along these lines:

```suggestion
                        <Menu.Item
                          component={Link}
                          to={commentsPath}
                          disabled={!commentsPath}
                          leftSection={<Icon name="add_comment" size={14} />}
                        >
                          {t`Comment`}
                        </Menu.Item>
```